test-ts/app/calendar.spec.ts

```diff
@@ -1,14 +1,89 @@
+import moment from 'moment-timezone';
 import Calendar from '@src/factory/calendar';
+import { UTCDate, LocalDate, MomentDate } from '@toast-ui/date';
+import { setDateConstructor } from '@src/time/timezone';
 
-describe('View/Calendar', () => {
-  it('Calendar view', () => {
-    const data = 1;
+describe('Calendar', () => {
+  describe('Timezone', () => {
+    beforeEach(() => {
+      MomentDate.setMoment(moment);
+    });
 
-    expect(data).toBe(1);
+    afterEach(() => {
+      setDateConstructor(LocalDate);
```

Comment:
Is this part intended to initialize with a local date after TC?

Reply:
Yes, it is.